bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ClientLibrariesImpl.java

```diff
@@ -235,20 +271,23 @@ private String getInline(LibraryType libraryType) {
         return output.toString();
     }
 
+    /**
+     * Get the clientlib categories from the list of component resourceTypes, using the defined
```

Comment:
I'd phrase it as: Gets the clientlib categories from the list of component resource types, filtered with the defined filter.

Reply:
Done in 000c479